tests/languages/wgsl/attributes_feature.test

```diff
@@ -0,0 +1,75 @@
+@notAnAttribute
+@align()
+@binding()
+@builtin()
+@const
+@group()
+@id()
+@interpolate()
+@invariant
+@location()
+@size()
+@workgroup_size()
+
+@vertex
+@fragment
+@compute
+
+----------------------------------------------------
+
+[
+	"@notAnAttribute\r\n@",
+	["attributes", "align"],
+	["punctuation", "("],
+	["punctuation", ")"],
+
+	"\r\n@",
+	["attributes", "binding"],
+	["punctuation", "("],
+	["punctuation", ")"],
+
+	"\r\n@",
+	["attributes", "builtin"],
+	["punctuation", "("],
+	["punctuation", ")"],
+
+	"\r\n@",
+	["attributes", "const"],
+
+	"\r\n@",
+	["attributes", "group"],
+	["punctuation", "("],
+	["punctuation", ")"],
+
+	"\r\n@",
+	["attributes", "id"],
+	["punctuation", "("],
+	["punctuation", ")"],
+
+	"\r\n@",
```

Comment:
Could we highlight the `@` as well in some way? Maybe as part of the attribute or as punctuation.